editor/src/utils/modifiers.ts

```diff
@@ -5,6 +5,8 @@ export type Modifiers = {
   shift: boolean
 }
 
+export const emptyModifiers: Modifiers = { alt: false, cmd: false, ctrl: false, shift: false }
```

Comment:
It's probably worth wrapping this in an `optionalDeepFreeze`

Reply:
done